enterprise/cypher/acceptance-spec-suite/src/test/resources/cypher/features/TemporalComparisonAcceptance.feature

```diff
@@ -183,16 +183,16 @@ Feature: TemporalComparisonAcceptance
               duration({years: 12, months:5, days: 14, hours:16, minutes: 12, seconds: 70}),
               duration({years: 12, months:5, days: 14, hours:16, minutes: 13, seconds: 10}),
               duration({years: 12, months:5, days: 13, hours:40, minutes: 13, seconds: 10})] as d
-      RETURN x>d, x<d, x>=d, x<=d, x=d
+      RETURN x=d
       """
     Then the result should be, in order:
-      | x>d   | x<d   | x>=d  | x<=d  | x=d    |
-      | null  | null  | null  | null  | false  |
-      | null  | null  | null  | null  | false  |
-      | null  | null  | null  | null  | false  |
-      | null  | null  | null  | null  | false  |
-      | null  | null  | null  | null  | false  |
-      | null  | null  | null  | null  | true   |
-      | null  | null  | null  | null  | true   |
-      | null  | null  | null  | null  | false  |
+      | x=d    |
```

Comment:
We seem to have lost a test that tested valid behavior, that is that inequality comparisons of durations should return null.

Reply:
Actually, semantic analysis now forbids comparing Durations. So if we know at compile time that we attempt to compare durations, we will get an error. If we _don't_ know at compile time, it will return null. I am gonna add a test for this behavior in `TemporalAcceptanceTest`

Reply:
When I looked at the new test it did not test the return null behaviour, but only the same throw error behaviour.